app/assets/javascripts/discourse/app/components/composer-messages.js

```diff
@@ -119,6 +122,43 @@ export default Component.extend({
     const composer = this.composer;
     if (composer.get("privateMessage")) {
       const recipients = composer.targetRecipientsArray;
+      const recipient_names = recipients
+        .filter((r) => r.type === "user")
+        .map(({ name }) => name);
+
+      if (
+        recipient_names.length > 0 &&
+        recipient_names.length !== _recipient_names.length &&
+        !recipient_names.every((v, i) => v === _recipient_names[i])
+      ) {
+        _recipient_names = recipient_names;
+
+        ajax(`/composer_messages/user_not_seen_in_a_while`, {
+          type: "GET",
+          data: {
+            usernames: recipient_names,
+          },
+        }).then((response) => {
+          if (
+            response.user_count > 0 &&
+            this.get("userNotSeenMessage") !== response.warning_message
+          ) {
+            this.set("userNotSeenMessage", response.warning_message);
+            this.messagesByTemplate["custom-body"] = undefined;
+            let title_key = "composer.user_not_seen_in_a_while_title.single";
+            if (response.user_count > 1) {
+              title_key = "composer.user_not_seen_in_a_while_title.multiple";
+            }
+            const message = composer.store.createRecord("composer-message", {
+              id: "user-not-seen",
+              templateName: "custom-body",
```

Comment:
```suggestion
              templateName: "education",
```

(Nothing wrong with your initial choice, but in a separate PR I just happened to delete the `custom-body` template because it's essentially the same as the `education` one.)